cost/turbonomics/rightsize_databases_recommendations/gcp/CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## v0.4
+
+- Unpublished policy from catalog since Turbonomic Rightsize Database does not support Google provider yet.
```

Comment:
I would remove this item since it doesn't really pertain to the functionality of the policy itself.